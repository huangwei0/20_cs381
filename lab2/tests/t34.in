debug off
stopwords off
load tracks.txt
preprocess
mcw ain't
mcw nobody
mcw got
mcw time
mcw for
mcw that