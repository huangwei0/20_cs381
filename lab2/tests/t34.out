no
knows
a
to
the
i
